console/backend/src/main/java/org/frankframework/console/controllers/socket/FrankApiWebSocketBase.java

```diff
@@ -67,23 +88,24 @@ protected final void propagateAuthenticationContext(String name) {
 	}
 
 	@Nullable
-	protected String compareAndUpdateResponse(RequestMessageBuilder builder) {
+	protected String compareAndUpdateResponse(RequestMessageBuilder builder, UUID target) {
 		final Message<?> response;
 		try {
-			response = gateway.sendSyncMessage(builder.build());
+			//TODO create a local cache and target specific members
```

Comment:
Can't MessageCacheStore expanded to work for this?

Reply:
```suggestion
```